front/components/sparkle/AppLayout.tsx

```diff
@@ -290,7 +290,7 @@ export default function AppLayout({
           </div>
           <div
             className={classNames(
-              "fixed left-0 right-0 top-0 z-30 flex h-16 flex-row lg:left-80",
+              "fixed left-0 left-12 right-0 top-0 z-30 flex h-16 flex-row lg:left-80",
```

Comment:
Maybe we want to remove the `left-0` then no ? Because I imagine only one of those 2 can apply ?

Reply:
Yep yep